frontend-react/src/content/product/sidenav.mdx

```diff
@@ -0,0 +1,7 @@
+<SideNav items={[
+    <SideNavItem href="/product/overview">Getting started</SideNavItem>,
```

Comment:
Is think correct? should it be` /getting-started` or should the label be "Overview of ReportStream"?

Reply:
just following the current sidenav hierarchy we have on the site. we can update as we add pages if that works for you?